packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -790,6 +790,16 @@ type MessagesType = {
   'modal.createaccountlearn.info3': "You can read more information about Stellar's minimum balance requirement at"
   'modal.createaccountlearn.link': 'Stellar.org'
   'modal.createaccountlearn.title': 'Stellar mimimum balance requirement.'
+
+  'modal.complete_profile.note': 'Your steps towards owning the future.'
+  'modal.complete_profile.finish_settings': 'Finish setting up your Blockchian.com account and start buying crypto today.'
```

Comment:
typo in blockchain

Reply:
Heh I reported this to design but I forgot to update it here, eagle eye!